CONTRIBUTING.md

```diff
@@ -1,61 +1,78 @@
 # Filing an issue
 
 ### Must read
-* If you aren't sure, you can ask on the [**IRC channel**](https://webchat.freenode.net/?channels=qtox) or read our [**wiki**](https://github.com/tux3/qTox/wiki) first.
+* If you aren't sure, you can ask on the
+  [**IRC channel**](https://webchat.freenode.net/?channels=qtox) or read our
```

Comment:
I think that the raw file is easier to understand when you leave it together.

Also, in GH's online editor and Atom, separating it breaks the code highlighting:

![without line break](https://cloud.githubusercontent.com/assets/6382342/14407055/8260cfec-febb-11e5-95b5-8d9470b4cef2.png)


![with line break](https://cloud.githubusercontent.com/assets/6382342/14407056/87de3f0e-febb-11e5-958a-32f97a0bd2ce.png)